.gitignore

```diff
@@ -10,6 +10,7 @@ venv
 node.json
 *.deployments.txt
 artifacts
+build
```

Comment:
We should keep the build/ directory not ignored when we add new solidity contracts, the output is put in the CONTRACT/build folder and we want them in the repository